Sources/Apodini/Semantic Model Builder/Model/Endpoint.swift

```diff
@@ -269,21 +221,21 @@ private struct AnyHashableEndpoint: Hashable, Equatable {
     }
 }
 
-
 extension EndpointsTreeNode {
```

Comment:
Would it make sense to move this extension (which is currently only used in Tests and which use case I'm not entirely sure of) into its own file as well?